web-frontend/src/pages/CreateAccount/StepTwo.tsx

```diff
@@ -17,7 +17,7 @@ export default function StepTwo() {
   const { mutate, isSuccess } = usePost<AccountObject>(`accounts/${nickname}`);
 
   if (isSuccess) {
-    navigate(routeFor(`${nickname}/account-create-step-three`));
+    navigate(routeFor('account-create-step-three'), { state: { nickname } });
```

Comment:
Non blocking: Soon or later we need to think how we can passa that across the app...